CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Our versioning strategy is as follows:
 
 ### 🎉 New Features & Improvements
 
+* `[create-sitecore-jss]``[angular]``[sitecore-jss-angular]` Introduce support for XM Cloud for Angular apps
+  * `[create-sitecore-jss]` Rework Angular initializer ([#1845](https://github.com/Sitecore/jss/pull/1845))
+
```

Comment:
Change log for angular would probably follow a bit different format, but that can be resolved when merging with https://github.com/Sitecore/jss/pull/1838

Reply:
@yavorsk please check the updated entry. Should be better I think?